scripts/base/init-bare.zeek

```diff
@@ -5840,6 +5840,11 @@ export {
 	##
 	## This currently has no effect for backend BROKER.
 	const log_serializer = Cluster::LOG_SERIALIZER_ZEEK_BIN_V1 &redef;
+
+	## Size of the WebSocket event queue for external clients.
+	##
+	## Only change this if you know what you're doing.
```

Comment:
This comment is really a cop-out, and we should at least say what effect this constant has. Also, I'd expect that changing most constants can have unexpected effects, so users should always validate their configurations -- there is no need to make that explicit everywhere.

Maybe something like this, with enough details:

```suggestion
	## Size of the WebSocket message queue for external clients.
	##
	## If there are more than `websocket_dispatcher_queue_size` messages to the client outstanding we will ...
```

Reply:
Fair!